contracts/OverlayToken.sol

```diff
@@ -20,20 +20,30 @@ contract OverlayToken is AccessControlEnumerable, ERC20("Overlay", "OVL") {
   }
 
   modifier onlyMinter() {
-    require(hasRole(MINTER_ROLE, msg.sender), "only minter");
+    require(hasRole(MINTER_ROLE, msg.sender), "ERC20: !minter");
     _;
   }
 
   modifier onlyBurner() {
-    require(hasRole(BURNER_ROLE, msg.sender), "only burner");
+    require(hasRole(BURNER_ROLE, msg.sender), "ERC20: !burner");
     _;
   }
 
   function mint(address _recipient, uint256 _amount) external onlyMinter {
       _mint(_recipient, _amount);
   }
 
-  function burn(address _account, uint256 _amount) external onlyBurner {
+  function burn(uint256 _amount) external onlyBurner {
+      _burn(msg.sender, _amount);
+  }
+
+  // See: OpenZeppelin Contracts v4.4.0 (token/ERC20/extensions/ERC20Burnable.sol)
+  function burnFrom(address _account, uint256 _amount) external onlyBurner {
```

Comment:
This appears to be contradictory to the advice from one of the reviewers that burn shall not be able to burn from any account.

We should consider if that is a desired system design where tokens are only burnt whence owned by a collateral manager at the time. Burn on transfer in. Burn on transfer out. 

A well designed transferBurn function could also satisfy these requirements.